server/src/main/java/timely/api/model/Metric.java

```diff
@@ -109,7 +122,9 @@ public void setVisibility(ColumnVisibility visibility) {
 
     public Mutation toMutation() {
         final byte[] row = rowCoder.encode(new ComparablePair<String, Long>(this.metric, this.timestamp));
-        final Value value = new Value(valueCoder.encode(this.value));
+        WRITE_BUFFER.get().clear();
+        WRITE_BUFFER.get().putDouble(this.value);
+        final Value value = new Value(WRITE_BUFFER.get().array());
```

Comment:
It's allocated with size Double.BYTES. I'm clearing it, and putting a Double into it. You are suggesting that it could still grow larger than Double.BYTES?